### ISSUE TITLE
Consider replacing PyCrypto with PyCryptodome

### ISSUE THREAD

**Issue by sbellem** [initc3/HoneyBadgerMPC#402]
> PyCryptodome can be used as: an almost drop-in replacement for the old PyCrypto library

PyCrytpo's development momentum seems to be paused since 2014; 58 pull requests currently open, 136 open issues. Currently fails for Python 3.8 because of its usage of `time.clock` -- trivial to fix though: https://github.com/dlitz/pycrypto/pull/296). 

All in all, PyCryptodome appears to be a better option.

_As a note_: PyCrytpodome has an implementation of Shamir's Secret Sharing: https://github.com/Legrandin/pycryptodome/blob/master/lib/Crypto/Protocol/SecretSharing.py.

**Comment by sanket1729** [comment]
Yes, we are already made this transition from pycryptodome to pycrypto for Crypto course this year.
